browser.js

```diff
@@ -240,7 +240,11 @@ function init() {
 
 	// link the theme if it was changed while the app was closed
 	if (process.platform === 'darwin') {
-		storage.set('darkMode', osxAppearance.isDarkMode());
+		const isDarkMode = osxAppearance.isDarkMode();
```

Comment:
Ignore my last comment.

This is using the system setting, not the `anatine` app setting. That's wrong, and in the end this whole block should be removed so we show whatever the last state of `anatine` we had selected, regardless of system settings.